### DISCUSSION TITLE
Saucedacity 1.2.1

### DISCUSSION THREAD

**Discussion by generic-pers0n** [saucedacity/saucedacity#68] [Announcements]
This is a small release intended to bring a few updates to libraries, macOS support, a very slight appearance change, and some project loading performance improvements backported from Audacity.

Note that this release **does not** bring anything from Tenacity. That won't happen until 1.3, which is making some steady changes. If you would like to try out release 1.3, you can check out https://nightly.link/saucedacity/saucedacity/workflows/cmake_build/main.

# Changes compared to 1.2
* Updated libmad submodule to 0.16.3
* **macOS specific changes**:
  * Fixed wrapper failing to launch Saucedacity. Turns out this wrapper hadn't been updated since the project was forked from upstream.
  * Updated app icon.
* Removed system border around our numeric text control.

## Changes from Audacity
* FFmpeg 5.0 is now officially supported
* Improved project loading performance by ~10x.

# Notes on macOS Binaries
* macOS is believed to work on macOS Intel. If you're on Apple Silicon, you will need to use Rosetta 2.
* **Saucedacity binaries for macOS are unsigned.** You will need to go to System Preferences > Security & Privacy > General and click "Open Anyway" if you attempted to run Saucedacity with Gatekeeper enabled.

<hr /><em>This discussion was created from the release <a href='https://github.com/saucedacity/saucedacity/releases/tag/v1.2.1'>Saucedacity 1.2.1</a>.</em>

**Comment by RockyMM** [comment]
Hi there.

I can confirm that Saucedacity 1.2.1 works on my Mac 👏 The installer works and the main executable works too.

However, there is a slight design issue 😅 

![image](https://user-images.githubusercontent.com/1906065/198115648-1cc63b79-93fc-4271-aa8f-4a3860771def.png)
I don't think white on white is looking nice. I would propose that the background has a slightly different tone, maybe a light gray, so that the Saucedacity icon stands out more.


Otherwise, so far, so good.

**Reply by generic-pers0n** [reply]
Ah yes, when you can only distinguish the 'S' 😆

Well, regrading should take care of it. That should hopefully happen not to far from now. Thanks for your feedback though!
